test/api/services/freshtrakApiService.test.ts

```diff
@@ -0,0 +1,39 @@
+import request from 'request-promise';
+import { getFreshTrakEvents } from '../../../api/services/freshtrakApiService';
+import { FRESHTRAK_API_HOST } from '../../../api/utils/constants';
+import freshtrakResponseDto from '../../../api/models/freshtrakAPI/freshtrakResponseDto';
+
+jest.mock('request-promise');
+
+describe('freshtrakApiService', () => {
+  describe('.getFreshTrakEvents', () => {
+    it('gets a freshtrak agency resource and returns a response object', async () => {
+      const get = jest.spyOn(request, 'get').mockResolvedValue(
```

Comment:
Since you only have one test it won't matter but usually setting up mocks go in a `beforeEach/All` block

Reply:
Seems like a good practice in case we need to add more tests here.